frontend/libs/studio-components/src/components/StudioDropdownMenu/StudioDropdownMenu.test.tsx

```diff
@@ -97,11 +97,16 @@ const renderTestDropdownMenu = () =>
         <StudioDropdownMenu.Item ref={group1Item2Ref}>{group1Item2Text}</StudioDropdownMenu.Item>
       </StudioDropdownMenu.Group>
       <StudioDropdownMenu.Group heading={group2Heading}>
-        <StudioDropdownMenu.Item icon={icon1}>{group2Item1Text}</StudioDropdownMenu.Item>
-        <StudioDropdownMenu.Item icon={icon2} iconPlacement='left'>
+        <StudioDropdownMenu.Item>
+          {icon1}
+          {group2Item1Text}
+        </StudioDropdownMenu.Item>
+        <StudioDropdownMenu.Item iconPlacement='left'>
+          {icon2}
           {group2Item2Text}
         </StudioDropdownMenu.Item>
-        <StudioDropdownMenu.Item icon={icon3} iconPlacement='right'>
+        <StudioDropdownMenu.Item iconPlacement='right'>
+          {icon3}
           {group2Item3Text}
         </StudioDropdownMenu.Item>
```

Comment:
Siden du har tatt høyde for endringen på `icon` i `StudioDropdownMenuItem`, er det vel ikke nødvendig å endre implementasjonen?

Reply:
Enig, her har jeg mikset litt. 
Men får en typescript error når jeg reverserer, "No overload matches". Det klager på at Element ikke er assignable til boolean & ReactNode. 
Jeg la til `as boolean & ReactNode` for å få bort feilmeldingen, men skjønner at det ikke er en optimal løsning. 
Usikker på hvor den får boolean fra. Har du noen tanker på hvordan jeg kan løse det?